net-mgmt/zabbix-agent/src/opnsense/mvc/app/controllers/OPNsense/ZabbixAgent/forms/settings.xml

```diff
@@ -0,0 +1,129 @@
+<form>
+    <tab id="zabbixagent-settings-main" description="Main Settings">
+        <field>
+            <id>zabbixagent.settings.main.enabled</id>
+            <label>Enabled</label>
+            <type>checkbox</type>
+            <help><![CDATA[Enable Zabbix Agent service.]]></help>
+        </field>
+        <field>
+            <id>zabbixagent.settings.main.hostname</id>
```

Comment:
so yes, why not use the system hostname (preferrably)

wrap the model as "zabbixagent.hostname" and then only sync "zabbix.settings"